Sources/ClientRuntime/Networking/Http/Middlewares/AuthSchemeMiddleware.swift

```diff
@@ -12,6 +12,7 @@ import class Smithy.Context
 import class SmithyHTTPAPI.SdkHttpRequestBuilder
```

Comment:
Import statements have been made granular & updated to match new organization. No further comments will be made for this type of change.